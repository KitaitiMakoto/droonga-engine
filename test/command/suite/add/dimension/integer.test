#@require-catalog-version 2
{
  "type": "add",
  "dataset": "Default",
  "body": {
    "table": "Products",
    "key": 1,
    "values": {
      "name": "Groonga"
    }
  }
}
{
  "type": "add",
  "dataset": "Default",
  "body": {
    "table": "Products",
    "key": 2,
    "values": {
      "name": "Droonga"
    }
  }
}
{
  "type": "add",
  "dataset": "Default",
  "body": {
    "table": "Products",
    "key": 3,
    "values": {
      "name": "Ruby"
    }
  }
}
{
  "type": "search",
  "dataset": "Default",
  "body": {
    "queries": {
      "products": {
        "source": "Products",
        "sortBy": ["_key"],
        "output": {
          "elements": ["records"],
          "attributes": ["_key", "name"],
          "limit": -1
        }
      }
    }
  }
}
