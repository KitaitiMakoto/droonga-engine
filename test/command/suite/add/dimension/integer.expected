{
  "inReplyTo": "request-id",
  "statusCode": 200,
  "type": "add.result",
  "body": true
}
{
  "inReplyTo": "request-id",
  "statusCode": 200,
  "type": "add.result",
  "body": true
}
{
  "inReplyTo": "request-id",
  "statusCode": 200,
  "type": "add.result",
  "body": true
}
{
  "inReplyTo": "request-id",
  "statusCode": 200,
  "type": "search.result",
  "body": {
    "products": {
      "records": [
        [
          1,
          "Groonga"
        ],
        [
          2,
          "Droonga"
        ],
        [
          3,
          "Ruby"
        ]
      ]
    }
  }
}
